bpf-sys/build.rs

```diff
@@ -28,6 +28,11 @@ pub mod headers {
 
 fn main() {
     println!("cargo:rustc-link-lib=static=bpf");
+    println!("cargo:rerun-if-changed=bcc/");
+    println!("cargo:rerun-if-changed=libbpf/");
+    println!("cargo:rerun-if-changed=libelf/");
+    println!("cargo:rerun-if-changed=bpfsys-musl.h");
+    println!("cargo:rerun-if-changed=libbpf_xdp.h");
 
```

Comment:
I don't think `rerun-if-changed=directory/` does the right thing, see https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargorerun-if-changedpath